lib/components/base/Pagination.vue

```diff
@@ -42,73 +42,63 @@
       :tabindex="page === pages[pages.length - 1] ? -1 : 0"
       class="right-arrow paginate has-icon"
       aria-label="Next Page"
-      :href="linkFunction(page + 1)"
+      :href="linkFunction?.(page + 1)"
```

Comment:
```suggestion
      :href="linkFunction(page + 1)"
```

Optional chaining can be avoided by following the suggestion below.